Makefile.common.mk

```diff
@@ -3,7 +3,7 @@
 # The original version of this file is located in the https://github.com/istio/common-files repo.
 # If you're looking at this file in a different repo and want to make a change, please go to the
 # common-files repo, make the change there and check it in. Then come back to this repo and run
-# "make updatecommon".
+# "make update-common".
 
 # Copyright 2019 Istio Authors
```

Comment:
should the copyright not precede the instructions?

Reply:
Let's discuss this in common-files since that's where this comes from.